.github/workflows/build.yml

```diff
@@ -25,14 +25,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # Node 14 matches the version of Node used by VS Code when this was
-      # written, but it should be updated when VS Code updates its Node version.
+      # Node 16 matches the version of Node used by VS Code when this was
+-     # written, but it should be updated when VS Code updates its Node version.
```

Comment:
I guess the added `-` is just a typo, isn't it?

Reply:
After https://github.com/microsoft/vscode-arduino/pull/1493/commits/fb9c4d8336da895bc4f6df8552e3f5a307d446cd there is one space missing, so the comments aren't aligned...

Reply:
Fixed in https://github.com/microsoft/vscode-arduino/pull/1493/commits/fb9c4d8336da895bc4f6df8552e3f5a307d446cd